<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuxiliaryAnalysisWidgetClass</class>
 <widget class="QWidget" name="AuxiliaryAnalysisWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1140</width>
    <height>697</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AuxiliaryAnalysisWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_12">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_11">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QTreeWidget" name="m_AuxiliaryAnalysisTtreeWid">
         <property name="maximumSize">
          <size>
           <width>240</width>
           <height>16777215</height>
          </size>
         </property>
         <attribute name="headerVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="headerHighlightSections">
          <bool>false</bool>
         </attribute>
         <attribute name="headerStretchLastSection">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string>安全特性参数分析</string>
          </property>
         </column>
         <item>
          <property name="text">
           <string>快速烤燃特性安全性分析</string>
          </property>
          <property name="checkState">
           <enum>Unchecked</enum>
          </property>
          <item>
           <property name="text">
            <string>计算值</string>
           </property>
           <property name="checkState">
            <enum>Unchecked</enum>
           </property>
          </item>
          <item>
           <property name="text">
            <string>标准值</string>
           </property>
           <property name="checkState">
            <enum>Unchecked</enum>
           </property>
          </item>
          <item>
           <property name="text">
            <string>权衡值</string>
           </property>
           <property name="checkState">
            <enum>Unchecked</enum>
           </property>
          </item>
         </item>
         <item>
          <property name="text">
           <string>慢速烤燃特性安全性分析</string>
          </property>
          <property name="checkState">
           <enum>Unchecked</enum>
          </property>
          <item>
           <property name="text">
            <string>计算值</string>
           </property>
           <property name="checkState">
            <enum>Unchecked</enum>
           </property>
          </item>
          <item>
           <property name="text">
            <string>标准值</string>
           </property>
           <property name="checkState">
            <enum>Unchecked</enum>
           </property>
          </item>
          <item>
           <property name="text">
            <string>权衡值</string>
           </property>
           <property name="checkState">
            <enum>Unchecked</enum>
           </property>
          </item>
         </item>
         <item>
          <property name="text">
           <string>枪击特性安全性分析</string>
          </property>
          <property name="checkState">
           <enum>Unchecked</enum>
          </property>
          <item>
           <property name="text">
            <string>计算值</string>
           </property>
           <property name="checkState">
            <enum>Unchecked</enum>
           </property>
          </item>
          <item>
           <property name="text">
            <string>标准值</string>
           </property>
           <property name="checkState">
            <enum>Unchecked</enum>
           </property>
          </item>
          <item>
           <property name="text">
            <string>权衡值</string>
           </property>
           <property name="checkState">
            <enum>Unchecked</enum>
           </property>
          </item>
         </item>
         <item>
          <property name="text">
           <string>射流冲击安全性分析</string>
          </property>
          <property name="checkState">
           <enum>Unchecked</enum>
          </property>
          <item>
           <property name="text">
            <string>计算值</string>
           </property>
           <property name="checkState">
            <enum>Unchecked</enum>
           </property>
          </item>
          <item>
           <property name="text">
            <string>标准值</string>
           </property>
           <property name="checkState">
            <enum>Unchecked</enum>
           </property>
          </item>
          <item>
           <property name="text">
            <string>权衡值</string>
           </property>
           <property name="checkState">
            <enum>Unchecked</enum>
           </property>
          </item>
         </item>
         <item>
          <property name="text">
           <string>破片撞击安全性分析</string>
          </property>
          <property name="checkState">
           <enum>Unchecked</enum>
          </property>
          <item>
           <property name="text">
            <string>计算值</string>
           </property>
           <property name="checkState">
            <enum>Unchecked</enum>
           </property>
          </item>
          <item>
           <property name="text">
            <string>标准值</string>
           </property>
           <property name="checkState">
            <enum>Unchecked</enum>
           </property>
          </item>
          <item>
           <property name="text">
            <string>权衡值</string>
           </property>
           <property name="checkState">
            <enum>Unchecked</enum>
           </property>
          </item>
         </item>
         <item>
          <property name="text">
           <string>跌落安全性分析</string>
          </property>
          <property name="checkState">
           <enum>Unchecked</enum>
          </property>
          <item>
           <property name="text">
            <string>计算值</string>
           </property>
           <property name="checkState">
            <enum>Unchecked</enum>
           </property>
          </item>
          <item>
           <property name="text">
            <string>标准值</string>
           </property>
           <property name="checkState">
            <enum>Unchecked</enum>
           </property>
          </item>
          <item>
           <property name="text">
            <string>权衡值</string>
           </property>
           <property name="checkState">
            <enum>Unchecked</enum>
           </property>
          </item>
         </item>
         <item>
          <property name="text">
           <string>爆炸冲击波安全性分析</string>
          </property>
          <property name="checkState">
           <enum>Unchecked</enum>
          </property>
          <item>
           <property name="text">
            <string>计算值</string>
           </property>
           <property name="checkState">
            <enum>Unchecked</enum>
           </property>
          </item>
          <item>
           <property name="text">
            <string>标准值</string>
           </property>
           <property name="checkState">
            <enum>Unchecked</enum>
           </property>
          </item>
          <item>
           <property name="text">
            <string>权衡值</string>
           </property>
           <property name="checkState">
            <enum>Unchecked</enum>
           </property>
          </item>
         </item>
         <item>
          <property name="text">
           <string>殉爆安全性分析</string>
          </property>
          <property name="checkState">
           <enum>Unchecked</enum>
          </property>
          <item>
           <property name="text">
            <string>计算值</string>
           </property>
           <property name="checkState">
            <enum>Unchecked</enum>
           </property>
          </item>
          <item>
           <property name="text">
            <string>标准值</string>
           </property>
           <property name="checkState">
            <enum>Unchecked</enum>
           </property>
          </item>
          <item>
           <property name="text">
            <string>权衡值</string>
           </property>
           <property name="checkState">
            <enum>Unchecked</enum>
           </property>
          </item>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="m_AuxiliaryAnalysisProgressBar">
         <property name="maximumSize">
          <size>
           <width>240</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="value">
          <number>24</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="tableWidget">
         <property name="maximumSize">
          <size>
           <width>240</width>
           <height>160</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="showGrid">
          <bool>true</bool>
         </property>
         <property name="gridStyle">
          <enum>Qt::SolidLine</enum>
         </property>
         <property name="sortingEnabled">
          <bool>false</bool>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <property name="cornerButtonEnabled">
          <bool>true</bool>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>99</number>
         </attribute>
         <attribute name="horizontalHeaderHighlightSections">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <row>
          <property name="text">
           <string>新建行</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>新建行</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>新建行</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>新建行</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>新建行</string>
          </property>
         </row>
         <column>
          <property name="text">
           <string>属性</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>值</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QWidget" name="m_chartWidget" native="true">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_10">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QWidget" name="widget_1" native="true">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QGraphicsView" name="graphicsView_1"/>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QWidget" name="widget_2" native="true">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QGraphicsView" name="graphicsView_2"/>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QWidget" name="widget_3" native="true">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QGraphicsView" name="graphicsView_3"/>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QWidget" name="widget_4" native="true">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QGraphicsView" name="graphicsView_4"/>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QWidget" name="widget_0" native="true">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QGraphicsView" name="graphicsView_0"/>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QWidget" name="widget_5" native="true">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QGraphicsView" name="graphicsView_5"/>
             </item>
            </layout>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QWidget" name="widget_6" native="true">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QGraphicsView" name="graphicsView_6"/>
             </item>
            </layout>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QWidget" name="widget_7" native="true">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_8">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QGraphicsView" name="graphicsView_7"/>
             </item>
            </layout>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QWidget" name="widget_8" native="true">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_9">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QGraphicsView" name="graphicsView_8"/>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
